src/utils/AutoId.tsx

```diff
@@ -0,0 +1,51 @@
+/** Based heavily on https://github.com/reach/reach-ui/tree/develop/packages/auto-id */
```

Comment:
Is this enough mention of the use of idea of reach-ui's Auto ID? 

Basically we have just few minor changes to it get it working in our setup and our own wrapper component `<AutoId>`.

Reply:
Might be sufficient as is. Could also be beneficial to add some explanation on why an how, not the whole story from the link.

Reply:
Added some explanation/reasoning as a comment.

Reply:
As this is so heavily based on Reach-UI implementation, I think we should include the Reach MIT Licence to our LICENCE file. Otherwise the implementation could have similar idea, but should be really our own. https://github.com/reach/reach-ui/blob/develop/LICENSE

Reply:
**reach-ui**'s MIT license is now added to our license.